sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py

```diff
@@ -126,9 +126,75 @@ def get_client_access_token(self, **kwargs: Any) -> JSON:
 
     get_client_access_token.metadata = {"url": "/api/hubs/{hub}/:generateToken"}  # type: ignore
 
+    @overload
+    def send_to_all(  # pylint: disable=inconsistent-return-statements
```

Comment:
Better to add these changes in CHANGELONG.md

Reply:
@msyyc Latest commit updated changelog.md and corrected types for parameter `content` in comments of `send*` methods. If it's OK, please help approve & merge.